### PR TITLE
drag and drop improvements

### PR DESCRIPTION
**Pull request by 3nids**
from https://github.com/qgis/QGIS/pull/34411 https://github.com/qgis/QGIS/pull/34433 and https://github.com/qgis/QGIS/pull/34479

### CODE REVIEW

File: source/docs/user_manual/working_with_vector/vector_properties.rst [hunk 1]
```diff
@@ -1992,7 +1992,7 @@ in figure_fields_form_.
 #. The next step is to assign the relevant fields to each container,
    by simple drag and drop. Groups and tabs can also be moved in the same way.
 #. At their creation time you can set properties of the containers
-   These, and other properties can be updated with a double-click - QGIS opens a form in
+   These, and other properties can be updated by selecting the item - QGIS displays a form in
```
Comment (havatv) [inline source/docs/user_manual/working_with_vector/vector_properties.rst]:
```suggestion
   These, and other properties can be updated by selecting the item.
   QGIS will the display a form where you can:
```

File: source/docs/user_manual/working_with_vector/vector_properties.rst [hunk 2]
```diff
@@ -1992,7 +1992,7 @@ in figure_fields_form_.
...
    which you can:
```
Comment (havatv) [inline source/docs/user_manual/working_with_vector/vector_properties.rst]:
```suggestion
```

**Comment by DelazJ** [comment]
@3nids I took the liberty of...
what does the cardinality field stand for? It only display "many to one relation" in the combobox here and i must admit that i don't use really advanced forms so, are there any other values possible? When?

**Comment by 3nids** [comment]
> @3nids I took the liberty of...

thanks a lot!

> what does the cardinality field stand for? It only display "many to one relation" in the combobox here and i must admit that i don't use really advanced forms so, are there any other values possible? When?

The cardinality allows you to chain the relations, hence N:M relations. If you have a join table (layer_a <= join_table => layer_b), layer_a and layer_b have a N:M relation.

**Pull request merged**